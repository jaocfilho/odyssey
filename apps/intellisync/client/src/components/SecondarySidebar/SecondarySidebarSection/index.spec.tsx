import { cleanup, render } from '@testing-library/react';
import { afterEach, describe, expect, it } from 'vitest';

import { SecondarySidebarSection } from '.';

describe('SecondarySidebarSection', () => {
  afterEach(() => {
    cleanup();
  });

  it('should render without crashing', () => {
    const { container } = render(
      <SecondarySidebarSection>SecondarySidebarSection</SecondarySidebarSection>
    );

    expect(container).toMatchSnapshot();
  });

  it('should not have border if it is the last section', () => {
    const { container } = render(
      <SecondarySidebarSection isLastSection>
        SecondarySidebarSection
      </SecondarySidebarSection>
    );

    expect(container).toMatchSnapshot();
  });
});
