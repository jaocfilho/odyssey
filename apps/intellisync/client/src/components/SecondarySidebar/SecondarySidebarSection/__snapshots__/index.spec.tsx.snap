// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SecondarySidebarSection > should not have border if it is the last section 1`] = `
<div>
  <div
    class="flex flex-col gap-4 py-4 px-2"
  >
    SecondarySidebarSection
  </div>
</div>
`;

exports[`SecondarySidebarSection > should render without crashing 1`] = `
<div>
  <div
    class="flex flex-col gap-4 py-4 px-2 border-b dark:border-zinc-50/10"
  >
    SecondarySidebarSection
  </div>
</div>
`;
